lib/danger/dangerfile.rb

```diff
@@ -42,6 +42,10 @@ def parse(path, contents = nil)
                     'you should turn off smart quotes in your editor of choice.'.red
       end
 
+      if contents.include?("puts")
+        puts "You used `puts` in your Dangerfile. To print out text to GitHub use `message` instead"
+      end
```

Comment:
I was thinking about just overwriting `puts` so that it's basically the same as `message`. I do this in `fastlane` already. What do you think @orta